stylesheets/style.css

```diff
@@ -0,0 +1,219 @@
+body {
+  padding: 0;
+  margin: 0;
```

Comment:
[OPTIONAL] It is **important** to remove the browser defaults. To achieve this, add these styles to the wildcard selector to select all elements and not just the body:

``` css
* {
    padding: 0;
    margin: 0;
}
```
NOTE: This may alter the looks of your project. You might have to add more styles to fix it.